x-pack/plugins/security_solution/common/license/license.ts

```diff
@@ -36,21 +36,21 @@ export class LicenseService {
     return this.observable;
   }
 
-  public isGoldPlus() {
+  public isGoldPlus(): boolean | undefined {
```

Comment:
Why `undefined` on the return type for these methods?  Why not just make them always return a boolean?

I assume the issue is the use of `?`, right? would that be fixed if you add a `false` to the end of the parentheses:

```typescript
return (
      this.licenseInformation?.isAvailable &&
      this.licenseInformation?.isActive &&
      this.licenseInformation?.hasAtLeast('gold')
    ) || false;
  }	
```